src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -150,11 +150,11 @@ class MulticopterPositionControl : public ModuleBase<MulticopterPositionControl>
 	int8_t _old_landing_gear_position{landing_gear_s::GEAR_KEEP};
 
 	DEFINE_PARAMETERS(
-		(ParamFloat<px4::params::MPC_TKO_RAMP_T>) _takeoff_ramp_time, /**< time constant for smooth takeoff ramp */
-		(ParamFloat<px4::params::MPC_Z_VEL_MAX_UP>) _vel_max_up,
-		(ParamFloat<px4::params::MPC_Z_VEL_MAX_DN>) _vel_max_down,
-		(ParamFloat<px4::params::MPC_LAND_SPEED>) _land_speed,
-		(ParamFloat<px4::params::MPC_TKO_SPEED>) _tko_speed,
+		(ParamFloat<px4::params::MPC_TKO_RAMP_T>) MPC_TKO_RAMP_T, /**< time constant for smooth takeoff ramp */
+		(ParamFloat<px4::params::MPC_Z_VEL_MAX_UP>) MPC_Z_VEL_MAX_UP,
+		(ParamFloat<px4::params::MPC_Z_VEL_MAX_DN>) MPC_Z_VEL_MAX_DN,
+		(ParamFloat<px4::params::MPC_LAND_SPEED>) MPC_LAND_SPEED,
+		(ParamFloat<px4::params::MPC_TKO_SPEED>) MPC_TKO_SPEED,
 		(ParamFloat<px4::params::MPC_LAND_ALT2>) MPC_LAND_ALT2, /**< downwards speed limited below this altitude */
```

Comment:
Hi @BazookaJoe1900 , I am a huge fan of getting naming standardized on one convention, thanks for doing this work.  I have a concern that by making them caps rather than `_some_class_member_var_name` (snake case with leading underscore for class member variables), it might be confusing in the code whether it is a class member var or the param intended to be used.
if you search the `Firmware/src/` directory for `DEFINE_PARAMETERS(` you will be able to see the convention followed in other instances of this pattern.  At a glance, the reset seem to adhere to the snakke case with leading underscore.
I hope this helps!

Reply:
@mcsauder, no problem. with your and @mhkabir direction I will change them to _snake_case convanation. I will do that to all parameters on all models

Reply:
Hi @BazookaJoe1900 , I understand where you are coming from.  Keep in mind that the usage of the `_class_member_variables` is limited in scope to that of `Class::_class_member_variable` while the `ACTUAL_PARAMETER_NAME` is effectively a globally accessible parameter within the [paramater list](https://dev.px4.io/en/advanced/parameter_reference.html).  I think keeping the two differentiated is important for the developer who might not want to not manipulate the values of the PARAMS directly but might need to manipulate a local copy within a scope limited class.